projects/ngrx.io/content/guide/effects/lifecycle.md

```diff
@@ -36,6 +38,58 @@ export class LogEffects {
 }
 </code-example>
 
+### Resubscribe on Error
+
+Starting with version 8, when an error happens in the effect's main stream it is
```

Comment:
I think we can leave out `Starting with version 8` as these are the docs for v8?

Reply:
Yes, but I'd rather highlight that it's a change from previous behavior. I find that beneficial. We can remove it by v9 :)